FrontBundle/EventSubscriber/CheckRoutingCacheViabilitySubscriber.php

```diff
@@ -39,12 +41,25 @@ public function checkCacheFileAndRefresh(GetResponseEvent $event)
             return;
         }
 
-        $cacheDir = $this->router->getOption('cache_dir');
+        $router = $this->router;
+        if ($this->router instanceof ChainRouter) {
+            foreach ($this->router->all() as $router) {
```

Comment:
As there is a chainRouter, you cannot access the option property for example, this way, i need to check through all the routers the one open orchestra one to be able to suppress the cache file created by him